translations/handlePlainTranslations.sh

```diff
@@ -12,7 +12,7 @@ gpg --list-keys
 git clone git@github.com:$1/$2 /app
 
 default_branch=$(git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@')
-versions="$default_branch $(git branch -r | grep -E "origin\/stable\-[0-9\.]+$" | cut -f2 -d"/")"
+versions="$default_branch $(git branch -r | grep -E "origin\/stable\-[0-9\.]+$" | cut -f2 -d"/" | head -n1)"
```

Comment:
At least for talk-android locally that's not working well:
```
git branch -r | grep -E "origin\/stable\-[0-9\.]+$" | cut -f2 -d"/" 
stable-20.1
stable-21.0
```

```
git branch -r | grep -E "origin\/stable\-[0-9\.]+$" | cut -f2 -d"/" | sort | head -n1
stable-20.1
```

So lets add a sort -r

```suggestion
versions="$default_branch $(git branch -r | grep -E "origin\/stable\-[0-9\.]+$" | cut -f2 -d"/" | sort -r | head -n1)"
```
```
git branch -r | grep -E "origin\/stable\-[0-9\.]+$" | cut -f2 -d"/" | sort -r | head -n1
stable-21.0
```